perfdash/config.go

```diff
@@ -80,6 +81,21 @@ var (
 				OutputFilePrefix: "ResourceUsageSummary",
 				Parser:           parseResourceUsageData,
 			}},
+			"LoadHighThroughputStartup": []TestDescription{{
```

Comment:
Let's name it with PodStartup prefix for consistency

Reply:
Done. Also, removed duplicated 'Startup'

Reply:
Actually, for density we are using `SaturationPodStartupLatency`. I will change new metrics to match this pattern